lib/rules/display-name.js

```diff
@@ -14,7 +14,9 @@ module.exports = Components.detect(function(context, components, utils) {
 
   var sourceCode = context.getSourceCode();
   var config = context.options[0] || {};
-  var acceptTranspilerName = config.acceptTranspilerName || false;
+  var acceptTranspilerName = config.hasOwnProperty('acceptTranspilerName') ?
```

Comment:
Yeah, I definitely like that better--I was just doing what @yannickcr had written in the plan. I'm going to be afk for the next few days starting soon, so I'll push another version and he can choose his favorite.